### ISSUE TITLE
not working for sbt 1.4.6

### ISSUE THREAD

**Issue by ThijsBroersen** [sbt/sbt-proguard#40]
When I try sbt-proguard on 1.4.x I get the following error:
```
(app / Proguard / proguardFilteredLibraries) java.lang.ClassCastException: class sbt.internal.inc.MappedVirtualFile cannot be cast to class java.io.File (sbt.internal.inc.MappedVirtualFile is in unnamed module of loader sbt.internal.MetaBuildLoader @bcec361; java.io.File is in module java.base of loader 'bootstrap')
```
Is this an sbt-proguard issue or Proguard?

**Comment by ScalaWilliam** [comment]
Hi, I am also getting this issue. It's most likely due to sbt-proguard building against SBT 0.13.x and not being up to date.

**Comment by ScalaWilliam** [comment]
@pvlugter are you able to look into this?

I tried to do fix this myself but came across multiple things that somebody specialised in SBT plug-ins would know.

The issue was not that it's building against SBT 0.13.x (there's cross SBT version support) but that for SBT 1.4 they have virtual file support, I don't know how to adapt it.

**Comment by alexarchambault** [comment]
FYI it's fixed manually with custom settings in [this file](https://github.com/coursier/coursier/blob/master/project/Settings.scala#L235) (look for "proguard" in it).

**Comment by apoorv-28** [comment]
> FYI it's fixed manually with custom settings in [this file](https://github.com/coursier/coursier/blob/master/project/Settings.scala#L235) (look for "proguard" in it).

This is throwing `404`

**Comment by SethTisue** [comment]
fixed by #42 (I'll open a separate ticket about getting the change published, though)
